Tasks/GoToolV0/task.json

```diff
@@ -33,7 +33,7 @@
             "name": "version",
             "type": "string",
             "label": "Version",
-            "defaultValue": "1.10",
+            "defaultValue": "1.19",
```

Comment:
Could you please specify the reason why you're changing default value?

Reply:
because Go 1.19 is the oldest supported version now